core/internals.h

```diff
@@ -74,8 +76,11 @@
 #define URI_REGISTRATION_SEGMENT_LEN    2
 #define URI_BOOTSTRAP_SEGMENT           "bs"
 #define URI_BOOTSTRAP_SEGMENT_LEN       2
+#define URI_DELETE_ALL_SEGMENT          "/"
+#define URI_DELETE_ALL_SEGMENT_LEN      1
```

Comment:
The bootstrap server is the only one able to access the security object. If you are adding a new LWM2M server to your network, you may want to configure your clients with this new server without factory-resetting them.